docs/development-setup.md

```diff
@@ -89,12 +109,11 @@ For IntelliJ, the following plugins are recomended:
 
 ### Backend Setup
 
-1. Install docker and docker-compose
-2. Setup docker-compose
+1. Install podman and podman-compose
+2. Setup podman-compose
 ```shell
-docker-compose rm
-docker-compose build
-docker-compose up --force-recreate
+podman machine init
+podman machine start && podman-compose up
```

Comment:
I would suggest to keep both options. I use docker for example. I can add a note how I made I'd work on mac

Reply:
Btw can you run backend tests with podman? we use TestContainers there, do they work fine with podman?

Reply:
> yes same for me i would like to keep docker as an alternative

Hmm, I am not exactly sure why we would want to keep both. The setup docs are only necessary/intended for new installations in which case it does not matter whether you use docker or podman. Since docker is proprietary now and is more complicated to set up, I think it is valid to only recommend one solution (being podman in this case). This does not mean that you should or even have to switch away from it if your current setup works fine.

> Btw can you run backend tests with podman? we use TestContainers there, do they work fine?

How can I run those?

Reply:
Should work with a small additional setup
https://podman-desktop.io/tutorial/testcontainers-with-podman
Maybe we need to include this to the instruction, if you could test that

Reply:
Did not have the motivation to get this up and running right now. I therefore opted to keep instructions for both docker and podman. This might be improved at a later point.